src/main/java/ch/njol/skript/expressions/ExprBrewingSlot.java

```diff
@@ -0,0 +1,140 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.effects.Delay;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.util.slot.InventorySlot;
+import ch.njol.skript.util.slot.Slot;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.BrewingStand;
+import org.bukkit.event.Event;
+import org.bukkit.event.block.BrewingStartEvent;
+import org.bukkit.event.inventory.BrewEvent;
+import org.bukkit.event.inventory.BrewingStandFuelEvent;
+import org.bukkit.inventory.BrewerInventory;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Brewing Stand Slot")
+@Description("A slot of a brewing stand, i.e. the 1st, 2nd, 3rd bottle slot, fuel or ingredient slot.")
+@Examples({
+	"set the brewing 1st bottle slot of {_block} to potion of water",
+	"clear the brewing second bottle slot of {_block}"
+})
+@Since("INSERT VERSION")
+public class ExprBrewingSlot extends PropertyExpression<Block, Slot> {
+
+	private enum BrewingSlot {
+		FIRST("[brewing [stand]] (first|1st) bottle"),
```

Comment:
`brewing first bottle` reads a bit weirdly to me. do we want to allow `'s` as in `brewing stand's first bottle`